pom.xml

```diff
@@ -74,7 +75,48 @@
       <groupId>com.datastax.spark</groupId>
       <artifactId>spark-cassandra-connector_${scala.main.version}</artifactId>
       <version>${connector.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>com.datastax.oss</groupId>
+	  <artifactId>java-driver-core-shaded</artifactId>
+	</exclusion>
+	<exclusion>
+	  <groupId>com.datastax.oss</groupId>
+	  <artifactId>java-driver-mapper-runtime</artifactId>
+	</exclusion>
+	<exclusion>
+	  <groupId>com.datastax.oss</groupId>
+	  <artifactId>java-driver-query-builder</artifactId>
+	</exclusion>
+	<exclusion>
+	  <groupId>com.datastax.oss</groupId>
+	  <artifactId>java-driver-shaded-guava</artifactId>
+	</exclusion>
+      </exclusions>
     </dependency>
+
+	<!-- Adding Java Driver latest version explicitly -->
+	<dependency>
+	  <groupId>com.datastax.oss</groupId>
+	  <artifactId>java-driver-core-shaded</artifactId>
+	  <version>${java-driver.version}</version>
+	</dependency>
+	<dependency>
+	  <groupId>com.datastax.oss</groupId>
+	  <artifactId>java-driver-mapper-runtime</artifactId>
+	  <version>${java-driver.version}</version>
+	</dependency>
+	<dependency>
+	  <groupId>com.datastax.oss</groupId>
+	  <artifactId>java-driver-query-builder</artifactId>
+	  <version>${java-driver.version}</version>
+	</dependency>
+	<!-- <dependency>
```

Comment:
This one was pulled earlier via the SCC, but actually not required. Keeping it here just in case commented out.